CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Mapbox welcomes participation and contributions from everyone.
 # main
 ## Bug fixes 🐞
 * Fix an issue when user subscribe sdk listeners multiple times, by changing CopyOnWriteArrayList to CopyOnWriteArraySet in the sdk to hold listeners. ([1183](https://github.com/mapbox/mapbox-maps-android/pull/1183))
+* Remove WeakRefernce from MapboxMap and Style. ([#1193](https://github.com/mapbox/mapbox-maps-android/pull/1193))
```

Comment:
```suggestion
* Remove WeakReference from MapboxMap and Style. ([#1193](https://github.com/mapbox/mapbox-maps-android/pull/1193))
```

Reply:
But also a more general question - do we need to add a note about that at all? Seems it's an implementation detail and users should not know about it.

Reply:
Think about it. Yes, this is an internal change, no need to expose it to users.